infrastructure/local/docker-compose.hlb.yml

```diff
@@ -1,23 +1,26 @@
 version: '3'
```

Comment:
Personal preference, but I don't think I would mind spelling out the ASE names fully: `hlb` -> `happy-life-bank`

Reply:
I think we could also remove `build-override.yml`?

Reply:
Did I forget to remove that? Yes, we can remove `build-override.yml`.

If we have it in separate folders, we won't need the ase's name in the file name anymore.